node_helper.js

```diff
@@ -11,10 +11,10 @@
 /* eslint-disable no-console */
 
 /**
- * @external node-fetch
- * @see https://www.npmjs.com/package/node-fetch
+ * @external logger
+ * @see https://github.com/MichMich/MagicMirror/blob/master/js/logger.js
```

Comment:
Update the URL to https://github.com/MagicMirrorOrg/MagicMirror/blob/master/js/logger.js